docs/azure/index-old.yml

```diff
@@ -0,0 +1,201 @@
+### YamlMime:Hub
+
+title: Azure for .NET developers
+summary: Find the information you need to use .NET on Azure
+brand: dotnet
+
+metadata:
+  title: Azure for .NET developers
+  description: Samples, tutorials, and education for .NET on Azure
+  ms.topic: hub-page
+  ms.prod: azure-dotnet
+  ms.date: 03/04/2020
+
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | whats-new 
+  items:
+    - itemType: quickstart
```

Comment:
How did we decide on this list? It seems a bit all over the place.

Reply:
Primarily feedback from developer advocates and historical telemetry.